scripts/get-envoy.bash

```diff
@@ -5,7 +5,7 @@ PATH="$PATH:$(go env GOPATH)/bin"
 export PATH
 
 _project_root="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)/.."
-_envoy_version=1.28.0
+_envoy_version=1.29.2
```

Comment:
Should probably mention in changelog

Reply:
changelogs are generated before release from tickets, based on labels. 
unless there's an `ignore-changelog` label set, everything is reflected in the changelog